jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -119,4 +125,51 @@ String getMainClass(JibExtension jibExtension) {
       throw new GradleException(ex.getMessage(), ex);
     }
   }
+
+  /**
+   * Returns an {@link ImageReference} parsed from the configured target image, or one of the form
+   * {@code project-name:project-version} if target image is not configured
+   *
+   * @param jibExtension the plugin configuration parameters to generate the name from
+   * @param gradleBuildLogger the logger used to notify users of the target image parameter
+   * @return an {@link ImageReference} parsed from the configured target image, or one of the form
+   *     {@code project-name:project-version} if target image is not configured
+   */
+  ImageReference getDockerTag(JibExtension jibExtension, GradleBuildLogger gradleBuildLogger)
```

Comment:
I think this method should explicitly reference that it's returning a tag for the target image. `getTargetDockerTag()`?  Or perhaps `getGeneratedTargetDockerTag()`?

(Is this really the right place? Might it be better on `JibExtension`?)

Reply:
I agree about a rename. I'm not sure if this kind of logic should go in `JibExtension`, since that's basically just for storing configuration parameters, but I'm not sure in this case since this is directly related to the target image configuration... what do you think @coollog?

Reply:
I agree there's probably a better place for this method than here, but it's probably not good to have it in `JibExtension` because methods on that are directly accessible to users in their `build.gradle` scripts when they configure `jib`.